Configuration/StandardSequences/python/AlCaRecoStreams_cff.py

```diff
@@ -272,6 +275,15 @@
 	dataTier = cms.untracked.string('ALCARECO')
 	)
 
+ALCARECOStreamTkAlZMuMuPA = cms.FilteredStream(
+        responsible = 'James Castle',
```

Comment:
suggestion, may be you can replace the ownership of ALCARECOStreamTkAlMuonIsolatedPA as well